packages/core/useMediaControls/index.ts

```diff
@@ -360,7 +360,10 @@ export function useMediaControls(target: MaybeRef<HTMLMediaElement | null | unde
       sources = [src]
 
     // Clear the sources
-    el.querySelectorAll('source').forEach(e => e.remove())
+    el.querySelectorAll('source').forEach((e) => {
+      e.removeEventListener('error', sourceErrorEvent.trigger)
```

Comment:
It looks like the event listener may not be deleted if a node references is stored somewhere. Therefore, I added this for prevention

Reply:
This is only called when `options.src` is changed though, right? We also want these listeners to be cleaned up when the component is unmounted.

Reply:
You are right. I forgot: 3